src/main/java/org/candlepin/subscriptions/tally/export/InstancesDataExporterService.java

```diff
@@ -26,6 +26,11 @@
 
 import com.redhat.swatch.configuration.registry.MetricId;
 import com.redhat.swatch.configuration.registry.ProductId;
+import com.redhat.swatch.export.DataExporterService;
+import com.redhat.swatch.export.DataMapperService;
+import com.redhat.swatch.export.ExportServiceException;
+import com.redhat.swatch.export.ExportServiceRequest;
+// NOTE(khowell): this couples our export implementation to the v1 REST API
```

Comment:
This comment needs to move down to line 53 now I believe.  Above `import org.candlepin.subscriptions.utilization.api.v1.model.ReportCategory;`

Reply:
yes, thanks for spotting.